lib/cisco_node_utils/interface.rb

```diff
@@ -379,7 +379,7 @@ def ipv4_pim_sparse_mode
 
     def ipv4_pim_sparse_mode=(state)
       check_switchport_disabled
-      Pim.feature_enable unless platform == :ios_xr || Pim.feature_enabled
+      Feature.pim_enable unless platform == :ios_xr || Feature.pim_enabled?
```

Comment:
You don't need the `|| Feature.pim_enabled?` since that's already checked by `Feature.pim_enable`